src/_main_/settings.py

```diff
@@ -137,16 +137,19 @@
 WSGI_APPLICATION = '_main_.wsgi.application'
 X_FRAME_OPTIONS = 'SAMEORIGIN'
 
-CSRF_COOKIE_SECURE = not DEBUG
-SESSION_COOKIE_SECURE = not DEBUG
-SESSION_COOKIE_HTTPONLY = not DEBUG
+CSRF_COOKIE_SECURE = True #not DEBUG
```

Comment:
Are these settings.py changes needed for something?

Reply:
I don't think we need those. I believe this is left over from my merge from the original GeoIP branch.